src/screens/Followings/FollowersScreen.js

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {BackHandler, Platform, StatusBar, StyleSheet, View} from 'react-native';
+import {useNavigation} from '@react-navigation/core';
+
+import Followings from '.';
+import {Context} from '../../context';
+import {setNavbarTitle, showHeaderProfile} from '../../context/actions/setMyProfileAction';
+import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
+import Search from '../DiscoveryScreenV2/elements/Search';
+import {getFollower} from '../../service/profile';
+import {Header} from '../../components';
+import {fonts} from '../../utils/fonts';
+
+function FollowersScreen() {
+  const navigation = useNavigation();
+  const [profileState, dispatchNavbar] = React.useContext(Context).profile;
+
+  const isAndroid = Platform.OS === 'android';
+
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [dataFollower, setDataFollower] = React.useState([]);
+  const [searchText, setSearchText] = React.useState('');
+
+  const fetchFollower = async (withLoading, text) => {
+    if (withLoading) setIsLoading(true);
+    const result = await getFollower(text);
+    if (result.code === 200) {
+      const newData = result.data.map((data) => ({
+        ...data,
+        name: data.user.username,
+        image: data.user.profile_pic_path,
+        description: null
+      }));
+      setDataFollower(newData);
+      if (withLoading) setIsLoading(false);
+    }
+  };
+
+  React.useEffect(() => {
+    if (searchText.length === 0) {
+      fetchFollower(true, searchText);
+    }
+  }, [searchText]);
+
+  React.useEffect(() => {
+    setNavbarTitle('Search Users', dispatchNavbar);
+  }, []);
+
+  const followingHeader = () => {
+    if ((Platform.OS === 'ios' && profileState.isShowHeader) || Platform.OS === 'android') {
+      return (
+        <Header
+          title="Who is following you"
+          // containerStyle={styles.header}
+          titleStyle={styles.headerTitle}
+          onPress={() => navigation.goBack()}
+          isCenter
+        />
+      );
+    }
+
+    return null;
+  };
+
+  const backPress = () => {
+    return false;
+  };
+
+  const settingBackhandleAndroid = () => {
+    BackHandler.addEventListener('hardwareBackPress', backPress);
+  };
+
+  const removeBackHandleAndroid = () => {
+    BackHandler.removeEventListener('hardwareBackPress', backPress);
+  };
+
+  React.useEffect(() => {
+    const unsubFocusListener = navigation.addListener('focus', () => {
+      showHeaderProfile(true, dispatchNavbar);
+    });
+    const unsubBlurListener = navigation.addListener('blur', () => {
+      showHeaderProfile(false, dispatchNavbar);
+    });
+    settingBackhandleAndroid();
+    return () => {
+      removeBackHandleAndroid();
+      unsubBlurListener();
+      unsubFocusListener();
+    };
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You are adding and removing event listeners for 'focus' and 'blur' events on every render of the component. This can lead to performance issues. Instead, these listeners should be added once when the component mounts and removed when it unmounts.

```diff
  React.useEffect(() => {
    const unsubFocusListener = navigation.addListener('focus', () => {
      showHeaderProfile(true, dispatchNavbar);
    });
    const unsubBlurListener = navigation.addListener('blur', () => {
      showHeaderProfile(false, dispatchNavbar);
    });
    settingBackhandleAndroid();
    return () => {
      removeBackHandleAndroid();
      unsubBlurListener();
      unsubFocusListener();
    };
  }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->